src/client/app/app.react.js

```diff
@@ -94,7 +96,7 @@ class App extends Component {
         <RouteHandler {...this.state} />
         <footer>
           <p>
-            made by <a href="https://twitter.com/steida">steida</a>
+            <FormattedHTMLMessage message={msg('app.madeByHtml')} />
```

Comment:
It's what I've read about speed difference of HTML templating engines vs. Markdown templating engines, never measured it myself. There can be some caching used to minimize this difference, but I'm not sure it would be necessary and/or good as another layer. BTW: I used very nice Markdown alternative and I just realized it's now for JS: https://github.com/mdipierro/markmin.js